src/Directory.Build.targets

```diff
@@ -26,4 +26,29 @@
 
   </Target>
 
+  <!-- API Compat -->
```

Comment:
Consider moving these to a seperate targets file if that is a convention you'd like to follow in this repo.

Reply:
I think we only have one for the src repo. But let me know if there is a better way.

---
In reply to: [279938766](https://github.com/dotnet/machinelearning/pull/3623#discussion_r279938766) [](ancestors = 279938766)